packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -170,11 +170,21 @@ export async function meterResponse(
       logger.info({ res }, 'Response sent')
     })
     .catch((err) => {
-      logger.error(ErrorMessage.UNKNOWN_ERROR)
-      logger.error(err)
+      logger.error({ err }, 'Caught error in outer endpoint handler')
+      if (!res.finished && !res.headersSent) {
```

Comment:
I'm still looking into exactly why typescript thinks the combiner/signer are using slightly different versions of express Responses. In the signer, it's slightly newer and has the `writableEnded` property available (while `finished` is deprecated), but not the case in the combiner.